docroot/themes/humsci/humsci_basic/readme.md

```diff
@@ -0,0 +1,68 @@
+# Humsci Basic (humsci_basic) Theme
+
+Humsci Basic (humsci_basic) is a theme that can be used as a base for all future H&S themes. We have pulled patterns and templates for Humsci Basic from Stanford Basic (stanford_basic) and Stanford University Humsci Theme (su_humsci_theme) Themes. Humsci Basic integrates selectively with Decanter Sass, styles, and patterns.
+
+## Requirements
+
+- Drupal 8.7.10
+- Node 10.15+
+- Decanter 5.1.0, soon to be upgraded to 6.0.0
+
+## Sub-themes
+
+There are currently 3 children sub-themes based on Humsci Basic. Themes themes reference built CSS files in Humsci Basic on a per-theme basis in their `{}.libraries.yml`.
+
+- Humsci Colorful (humsci_colorful)
+- Humsci Traditional (humsci_traditional)
+- Humsci Airy (humsci_airy)
+
+## Getting Started
+
+This theme contains its own node module dependencies and build system which is separate from the root project. All commands should be run from this theme's directory.
+
+- `npm install` - Install all node dependencies
+
+## Commits and Merging PRs
+
+### Commit Messages
+
+We are using a conventional commit message format, e.g.:
+
+`feat(STN-XX): descriptive message shorter than 80 chars`
+`docs(STN-XXX): descriptive message shorter than 80 chars`
+
+https://www.conventionalcommits.org/en/v1.0.0-beta.2/
+
+## Green Button Merging
+
+After receiving a review and getting a PR approved, we do green-button merges for our PRs ("Rebase and Merge") because Github includes a link to the PR in our commit message header.
+
+## Builds 
+
+Frontend assets are built using the Grunt task runner, but are run using npm scripts as shortcuts. Assets are compiled to a `dist/` directory with a stylesheet generated for each sub-theme.
+
+- `npm run build:sass` - Compile Sass for production
+- `npm run watch` - Compile a CSS build and watch for changes in the existing `.scss` files
+
+## Testing
+
+- `npm test` - Run linting and Sass True tests
```

Comment:
we also need to figure out how to run and make our own behat tests, and unit tests for modules